### DISCUSSION TITLE
Image not opening in the middle

### DISCUSSION THREAD

**Discussion by Destroy666x** [dimsemenov/PhotoSwipe#2109] [Q&A] [answered]
I'm using Phoca Gallery Joomla plugin that uses 4.1.3 version of this. There's a really weird problem where on e.g. this page: https://lenkraddesign.eu/oferta/dywaniki-samochodowe it results in small popup that only spawns on last instance of a gallery group.

But e.g. here: https://lenkraddesign.eu/galeria/dywaniki-samochodowe/dywaniki-welurowe/dywaniki-premium it opens fully correctly.

It also started occuring randomly. Without any errors in the console. I asked the developer of the plugin but so far this led to nothing. So trying my luck here. Any idea what even could cause something like this? As not to sure how to debug it, looks like a bug in either the gallery plugin or that Photoswipe version.

**Comment by dimsemenov** [comment]
In the old version, it depends on where the PhotoSwipe root HTML element is placed, to make it cover the screen - generally, it should be close to the root html node (if it's not - styles of parent elements should be adjusted). I don't think there is any easy way to fix it without coding knowledge.

**Reply by Destroy666x** [reply]
I do have coding knowledge, I'm working on the site. I don't have Joomla knowledge, as it's often a complete mess where you don't know where to look for anything. So thanks for a hint. Any specific classes/IDs I should search for?

**Reply by dimsemenov** [reply, marked as answer]
Check where `<div class="pswp">` element is added, and try to move it closer to the root document node, or fix its parent styles so it can overlap page content.

**Reply by Destroy666x** [reply]
The fix was to remove some animation-related class from parent div that added `visibility: none`, just on the last divs where the `pwsp` div spawned as a child. Thanks again for leading me onto right track!
